src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2968,15 +3003,41 @@ void AutomationInstrumentClipView::handleSinglePadPress(ModelStackWithTimelineCo
 				effectiveLength = clip->loopLength;
 			}
 
```

Comment:
adjusted the peek value code while in pad selection mode

when a long press is selected, the peek value at the start position should be the value at the very first node
when a long press is selected, the peek value at the end position should be the very at the very last node
when a long press is selected, the peek value at positions between start and end will be the value at the middle node position

for non-long press selections, the peek value is always the middle node value